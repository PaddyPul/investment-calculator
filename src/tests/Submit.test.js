import React from 'react';
import { Submit } from '../containers/Submit.js'; // importing unconnected component
import renderer from 'react-test-renderer';

test('Submit container rendering without issues', () => {
  const component = renderer.create(
    <Submit
      amount='2000'
      date='1'
      type='Bitcoin'
      label='Simular com o outro investimento' />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
