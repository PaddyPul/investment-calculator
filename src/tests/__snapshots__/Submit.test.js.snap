// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Submit container rendering without issues 1`] = `
<form
  onSubmit={[Function]}
>
  <button
    className="btn btn-primary submit__button"
    type="submit"
  >
    Simular com o outro investimento
  </button>
</form>
`;
